src/spec.md

```diff
@@ -6,14 +6,16 @@
 - [1. Introduction](#Introduction)
 - [2. Class Overview](#Class_Overview)
 - [3. EthStorage Class](#EthStorage)
-    - Constructor
+    - Static Methods
```

Comment:
The readme.md should also be updated, and it may need have a reference for the spec in the readme.

Reply:
I also suggest to move the spec.md to the root path of the repo